src/lib/day-calendar/day-calendar.component.ts

```diff
@@ -253,13 +257,57 @@ export class DayCalendarComponent implements OnInit, OnChanges, ControlValueAcce
     this.onLeftNav.emit({from, to});
   }
 
+  onLeftSecondaryNavClick(): void {
```

Comment:
please move this logic into the service, this way it much easier to read the component code and set unit tests to the service.

Reply:
Done